python/pyspark/sql/functions.py

```diff
@@ -147,7 +173,7 @@ def _():
 # math functions that take two arguments as input
 _binary_mathfunctions = {
     'atan2': 'Returns the angle theta from the conversion of rectangular coordinates (x, y) to' +
-             'polar coordinates (r, theta).',
+             'polar coordinates (r, theta). Units in radians.',
```

Comment:
I think we should ...

1. copy and paste `Units in radians.` to both https://github.com/apache/spark/blob/e5387018e76a9af1318e78c4133ee68232e6a159/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L1342 and https://github.com/apache/spark/blob/8965fe764a4218d944938aa4828072f1ad9dbda7/R/pkg/R/functions.R#L2030 just to be consistent

2. Use Python specific notation to describe this here for now.

3. Just revert this change for now.